docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+---
+version: "2.1"
+services:
+  mariadb:
+    image: lscr.io/linuxserver/mariadb:latest
+    container_name: mariadb
+    environment:
+      - PUID=501
+      - PGID=20
+      - MYSQL_ROOT_PASSWORD=Password123
+      - TZ=Europe/London
+    volumes:
+      - ./database:/config
+    ports:
+      - 4306:3306
+    restart: unless-stopped
+
+  baseball:
+    build:
+      context: .
+      dockerfile: dockerfile
+    image: baseball
+    volumes:
+      - ./baseball.sql:/App/baseball.sql
```

Comment:
I did this by copying the file in using a COPY command in the dockerfile. But this way seems to work too. One thing to note is that you're putting it in a folder called /App, but the working directory in the container is /app (lower case).